english/racquetball/allequpiment.md

```diff
@@ -0,0 +1 @@
+You beat the holographic opponent, and are pleased you've found a new place to play!
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.